container.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
The name "container.sh" could be improved, especially since this looks to be osx-specific. Maybe `mac-bldr.sh` ?